torchvision/transforms/functional.py

```diff
@@ -220,9 +220,10 @@ def pad(img, padding, fill=0):
             on left/right and top/bottom respectively. If a tuple of length 4 is provided
             this is the padding for the left, top, right and bottom borders
             respectively.
-        fill: Pixel fill value. Default is 0. If a tuple of
+        fill: Pixel fill value for constant fill. Default is 0. If a tuple of
             length 3, it is used to fill R, G, B channels respectively.
-
+        padding_mode: Type of padding. Should be: constant, edge, reflect or symmetric.
```

Comment:
Can you expand on the differences between `reflect` and `symmetric`? From looking at the images, it was not clear to me what were the differences.